tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/tuples/TupleAccessExprTest.java

```diff
@@ -38,10 +41,12 @@
 public class TupleAccessExprTest {
 
     private CompileResult compileResult;
+    private CompileResult compileNegativeResult;
 
     @BeforeClass
     public void setup() {
         compileResult = BCompileUtil.compile("test-src/types/tuples/tuple_access_expr.bal");
+        compileNegativeResult = BCompileUtil.compile("test-src/types/tuples/tuple_with_no_member_test.bal");
```

Comment:
Is the correct place for this test? Shouldn't this go in foreach tests? Where do we have the other tuple foreach tests?

Reply:
I will update that